### PR TITLE
Sean hockin

### PR DESCRIPTION
**Pull request by shockin779**
Almost done

### CODE REVIEW

**Comment by shockin779** [comment]
Completed with all stretch goals

File: challenges/objects-arrays.js [hunk 1]
```diff
@@ -97,24 +132,24 @@ The zoos need a list of all their animal's names (names only, not scientific) co
...
 
 /* Request 4: .reduce() 
 
 The zoos need to know their total animal population across the United States.  Find the total population from all the zoos using the .reduce() method.
 
 */
-const populationTotal = 0;
-console.log(populationTotal);
+const populationTotal = zooAnimals.reduce((acc, val) => acc += val.population, 0);
+console.log(`Total population accross the United States: ${populationTotal}`);
```
Comment (Dilstom) [inline challenges/objects-arrays.js]: objects-arrays - Done!

File: challenges/prototypes.js [hunk 2]
```diff
@@ -6,28 +6,41 @@
...
   Add properties and values of length: 4, width: 5, and height: 5 to cuboid.   
 */
 
+const cuboid = new CuboidMaker(4, 5, 5);
+
 // Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+console.log(cuboid.volume()); // 100
+console.log(cuboid.surfaceArea()); // 130
```
Comment (Dilstom) [inline challenges/prototypes.js]: prototypes - Done!

**Comment by Dilstom** [comment]
perfect as usual, Sean!

**Pull request merged**